crates/chia-protocol/Cargo.toml

```diff
@@ -5,8 +5,8 @@ edition = "2021"
 license = "Apache-2.0"
 description = "Chia network protocol message types"
 authors = ["Arvid Norberg <arvid@chia.net>"]
-homepage = "https://github.com/Chia-Network/chia_rs/chia-protocol/"
-repository = "https://github.com/Chia-Network/chia_rs/chia-protocol/"
+homepage = "https://github.com/Chia-Network/chia_rs"
+repository = "https://github.com/Chia-Network/chia_rs"
```

Comment:
An alternative fix would be to point it to: `https://github.com/Chia-Network/chia_rs/tree/main/chia-protocol`. I don't have a strong opinion though.

Reply:
I think it's typical for it to point to the repository's root, like in the following:

- https://github.com/bevyengine/bevy/blob/main/crates/bevy_audio/Cargo.toml#L7
- https://github.com/tokio-rs/tokio/blob/master/tokio-macros/Cargo.toml#L12
- https://github.com/rwf2/Rocket/blob/master/core/codegen/Cargo.toml#L8
- https://github.com/diesel-rs/diesel/blob/master/diesel_cli/Cargo.toml#L9